moduleframework/tools/helpmd_lint.py

```diff
@@ -57,11 +58,15 @@ def test_helpmd_image_name(self):
         container_name = self.dp.get_docker_specific_env("NAME=")
         if container_name:
             self.assertTrue(self.helpmd.get_image_name(container_name[0].split('=')[1]))
+        else:
+            self.cancel()
```

Comment:
Why cancel? If container_name does not exist, then it should fail.

Reply:
But it does not fails anyhow and mark test as PASS, although this testcase does not test anything and this is very bad.
This improves behavior, at least that it is now canceled, in case "if container_name" is not true instead of PASS.